src/components/UploadingJobsDisplay/components/FileItem/FileItem.js

```diff
@@ -10,93 +10,91 @@ import { getFileItemMeta } from './getFileItemMeta';
 
 import css from './FileItem.css';
 
-export class FileItem extends PureComponent {
-  static propTypes = {
-    uiKey: PropTypes.string.isRequired,
-    name: PropTypes.string.isRequired,
-    size: PropTypes.number.isRequired,
-    isSnapshotMode: PropTypes.bool,
-    status: PropTypes.string,
-    uploadedValue: PropTypes.number,
-    errorMsgTranslationID: PropTypes.string,
-    uploadedDate: PropTypes.string,
-    loading: PropTypes.bool,
-    onDelete: PropTypes.func,
-    onUndoDelete: PropTypes.func,
-  };
-
-  static defaultProps = {
-    isSnapshotMode: false,
-    status: FILE_STATUSES.UPLOADING,
-    uploadedValue: 0,
-    errorMsgTranslationID: 'upload.invalid',
-    uploadedDate: null,
-    loading: false,
-    onDelete: noop,
-    onUndoDelete: noop,
-  };
-
-  deleteFile = () => {
+export const FileItem = memo(props => {
+  const deleteFile = () => {
     const {
       uiKey,
       status,
       onDelete,
-    } = this.props;
+    } = props;
```

Comment:
Consider destructuring all props at the top of the component. Such an approach will make functions shorter.
```suggestion
    const deleteFile = () => onDelete(uiKey, status);
```

Reply:
Makes sense. Done.